<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DocumentAdd</class>
 <widget class="QDialog" name="DocumentAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>355</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>411</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>Добавить новый документ</string>
   </property>
   <widget class="QPushButton" name="ButtonToAddDocument">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить документ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Рейтинги вводить через пробел без запятых</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>391</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Введите документ</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="document"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Номер id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="document_id_string"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Статус документа</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="status_string"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Список рейтингов</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="rating_string"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
